Parser/asdl_c.py

```diff
@@ -1279,8 +1314,15 @@ def generate_module_def(f, mod):
         visitor.visit(mod)
         visitor_list.add(visitor)
 
-    state_strings = set(["__dict__", "_attributes", "_fields", "__module__", "_ast"])
-    module_state = set(["__dict__", "_attributes", "_fields", "__module__", "_ast"])
+    state_strings = {
```

Comment:
Oh-ho-ho, this file was intended to supported compatibility with very old Python versions, which did not supported set literals. I think it can be dropped now.

Reply:
Cool. I wasn't aware this file should support that old (which I also didn't even know which version sets added) files.